src/components/zones/ZonesList.tsx

```diff
@@ -115,18 +105,19 @@ export default function ZonesList() {
               </Flex>
             </AccordionItemTitleCustom>
             <AccordionPanel p={0}>
-              <Accordion allowToggle index={[openedZoneIndex]}>
-                {zones.map((z) => {
+              <Accordion allowToggle>
+                {drawnZones.map((z, i) => {
                   return (
-                    <>
-                      <AccordionItem key={z.id} border="none">
+                    <Fragment key={`${z.id}_${i}`}>
```

Comment:
why do we need the index in the key? It sounds counter productive since if we change the array, all indexes will change and it will force redraw of the full list. z.id should be enough probably? If not we may need to improve zone.id to be unique

Reply:
I added the index to redraw the box in this scenario : 
- the zones are automatically extracted
- the user moves/adjusts some zones (change x, y, width or height)
- the user clicks on "Reset all"
- the zones return to their initial position/dimensions
  
  
  I played with the zones and the reset all button was not resetting them but we can remove this functionality, what do you think @slecorne?

Reply:
I think @audreyhuguenin may have worked on the reset all zones in another PR, we can maybe check again when both PR merged